code/script.js

```diff
@@ -13,35 +16,133 @@ const showMessage = (message, sender) => {
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/smileyface.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/Food4cure.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  // This little thing makes the chat scroll to the last message 
+  //when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here 
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
+  showMessage(`Hi there, What's your name?`, 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+//Question 1
+const handleNameInput = event => {
+  event.preventDefault()
+  const userName = nameInput.value
+  nameInput.value = ''
+  showMessage(userName, 'user')
+  setTimeout(() => askForKnowledge(userName), 1000)
+}
+
+//Question 2
+const askForKnowledge = userName => {
+  showMessage(`Lovely to meet you ${userName}! What would you like to learn about today?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id='zincButton'>Zinc</button>
+  <button id='proteinButton'>Protein</button>
+  <button id='vitaminDButton'>VitaminD</button>
+`
```

Comment:
This could be indented for better readability:
```
  inputWrapper.innerHTML = `
    <button id='zincButton'>Zinc</button>
    <button id='proteinButton'>Protein</button>
    <button id='vitaminDButton'>VitaminD</button>
  `
```